docs/List.md

```diff
@@ -297,114 +297,56 @@ Bulk action button components receive several props allowing them to perform the
 * `filterValues`: the filter values. This can be useful if you want to apply your action on all items matching the filter.
 * `selectedIds`: the identifiers of the currently selected items.
 
-Here is an example leveraging the `useUpdateMany` hook, which sets the `views` property of all posts to `0`:
+Here is an example of `BulkUpdateButton` usage, which sets the `views` property of all posts to `0`:
 
 ```jsx
 // in ./ResetViewsButton.js
 import * as React from "react";
-import {
-    Button,
-    useUpdateMany,
-    useRefresh,
-    useNotify,
-    useUnselectAll,
-} from 'react-admin';
 import { VisibilityOff } from '@material-ui/icons';
+import { BulkUpdateButton } from 'react-admin';
 
-const ResetViewsButton = ({ selectedIds }) => {
```

Comment:
The new button is great, but the documentation also served as an example to write a custom BulkActionButton. Could you find a way to keep that?